### ISSUE TITLE
protontricks crash after game selection window

### ISSUE THREAD

**Issue by Frambooisier** [Matoking/protontricks#235]
**Describe the bug**
I start protontricks via flatpak with `run com.github.Matoking.protontricks --gui` command, first window appears where I can select game to be configured, after selection and confirmation the window disappears and no new window or message appears. In terminal 

**To Reproduce**
1. Run command `run com.github.Matoking.protontricks --gui`
2. select game title in the protontricks window
3.  crash, read the info from terminal

**Expected behavior**
if I start protontricks with --gui I expect to see first window with  selection of a game, then after that I expect to see another window where I can select what I want to do with selected game's prefix

**System (please complete the following information):**
 - Distro: 
Operating System: openSUSE Tumbleweed 20230617
KDE Plasma Version: 5.27.5
KDE Frameworks Version: 5.107.0
Qt Version: 5.15.10
Kernel Version: 6.3.7-1-default (64-bit)
Graphics Platform: X11
Processors: 20 × 12th Gen Intel® Core™ i7-12700K
Memory: 15,4 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 3070/PCIe/SSE2
Manufacturer: Micro-Star International Co., Ltd.
Product Name: MS-7D46
System Version: 1.0

 - Protontricks installation method: flatpak
 - Protontricks version: 1.10.3
 - Steam version: no beta, standard up-to-date 

protontricks terminal output with -v flag
https://gist.github.com/Frambooisier/0763abd948208f0fe95bb3d6c92e9b8d

**Comment by Frambooisier** [comment]
yes `pl_PL.utf8` is listed with `locale -a` command, on OpenSuse TW Im missing `locale-gen` command. My main system language is english and `/etc/locale.conf` only has `LANG=en_US.UTF-8` but variable `$LANG`  outputs `LANG=pl_PL.UTF-8`. I changed temporarly `$LANG` to `LANG=en_US.UTF-8` and it did fix the problem, now I will add `LANG=pl_PL.UTF-8` to `/etc/locale.conf` to see if it solves the problem permanently.

side note it seems suse use `localedef` so `locale-gen` is not present.

**Comment by Matoking** [comment]
Closing old issue, open new thread if this is still a problem.

**Comment by Frambooisier** [comment]
Sure thing, after your response I think I solved the problem with your knowledge. Thanks!
